chart/templates/_helpers.tpl

```diff
@@ -26,6 +26,25 @@ heritage: {{.Release.Service }}
 release: {{.Release.Name }}
 {{- end }}
 
+{{/*
+Windows cluster will add default taint for linux nodes, 
+add below linux tolerations to workloads could be scheduled to those linux nodes
+*/}}
+{{- define "linux-node-tolerations" -}}
+- key: "cattle.io/os"
+  value: "linux"
+  effect: "NoSchedule"
+  operator: "Equal"
+{{- end -}}
+
+{{- define "linux-node-selector" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+beta.kubernetes.io/os: linux
+{{- else -}}
+kubernetes.io/os: linux
+{{- end -}}
+{{- end -}}
+
```

Comment:
Are these used anywhere ?
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/deployment.yaml#L46-L47
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/deployment.yaml#L56
* https://github.com/rancher/charts/blob/dev-v2.5-source/packages/rancher-backup/charts/templates/_helpers.tpl#L12-L18

Reply:
Updated to use them